src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Section/MicrosoftTeamsSectionElementInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ */
+interface MicrosoftTeamsSectionElementInterface
```

Comment:
This interface is never really used. We use it twice in a type hint but we still check the class.

I suggest removing this.

Reply:
You are right, but we can keep it for consistency and extensibility.
We have:
* Action
* * ActionElement
* * ActionInput
* Section
* * SectionElement

All of them have a marker interface, which is ok for me 👍